app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/exemptionandlosses/further_return_in_year_losses.scala.html

```diff
@@ -0,0 +1,137 @@
+    backLink: Call,
+    isATrust: Boolean,
+    representativeType: Option[RepresentativeType],
+    isFurtherReturn: Option[Boolean]
+)(implicit request: RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
+
+@key = @{ "inYearLosses" }
+@valueKey = @{ "inYearLossesValue" }
+@isAgent = @{ request.userType.contains(UserType.Agent) }
+
+@furtherReturnKey = @{ if(isFurtherReturn.contains(true)) ".furtherReturn" else "" }
+
+@userKey = @{
+    representativeType match {
+      case Some(PersonalRepresentative) => ".personalRep"
+      case Some(PersonalRepresentativeInPeriodOfAdmin) if(!isAgent) => ".personalRepInPeriodOfAdmin"
+      case Some(PersonalRepresentativeInPeriodOfAdmin) if(isAgent) => ".personalRepInPeriodOfAdmin.agent"
+      case Some(Capacitor) => ".capacitor"
+      case None =>
+          if (isAgent) ".agent"
+          else if (isATrust) ".trust"
+          else ""
+  }
+}
+
+@title = @{messages(s"$key$furtherReturnKey.main$userKey.title")}
+@label = @{Html(s"""<h2 class="heading-medium">${messages(s"inYearLosses$furtherReturnKey$userKey.title")}</h2>""")}
+
+@hasErrors = @{form.hasErrors || form.hasGlobalErrors}
+@options = @{List(
+    RadioOption(
+        label = messages(s"generic.yes"),
+        content = Some(unitInput(
+            fieldId = valueKey,
+            value = form.data.getOrElse(valueKey, ""),
+            label = Html(messages(s"$valueKey$userKey.label")),
+            formError = form.error(valueKey),
+            hasErrors = form.error(valueKey).nonEmpty,
+            autoComplete = AutoCompleteType.On,
+            inputClass = Some("small"),
+            aspect = "left",
+            unit = "£",
+            assistiveText = messages("unit-input.assistive.pounds")
+        )),
+        optionHelpText = None
+    ),
+    RadioOption(
+        label = messages(s"generic.no"),
+        content = None,
+        optionHelpText = None
+    )
+
+)}
+@selectedIndex = @{
+    form.data.get(key).map({
+        case "0" => 0
+        case _ => 1
+    })
+}
+
+@mainTemplate(title = title, userType = request.userType, hasErrors = hasErrors, withSignOutLink = true) {
+
+    @backLinkComponent(backLink)
+
+    @if(hasErrors) {
+        @errorSummary(form, None, userKey)
+    }
+
+    @caption(messages("exemptionAndLosses.caption"))
+    @pageHeading(title)
+
+    <p>@messages(s"$key$furtherReturnKey$userKey.helpText1")</p>
+    <p>@messages(s"$key$furtherReturnKey$userKey.helpText2")</p>
+
+    @formWrapper(routes.ExemptionAndLossesController.inYearLossesSubmit(), 'novalidate -> "novalidate") {
+        @radioGroupConditional(
+            fieldId = key,
+            label = label,
+            selected = selectedIndex,
+            error = form.error(key),
+            options = options,
+            helpText = None,
+            customErrorKey = Some(s"$key$userKey")
+        )
+
+        <p>@Html(messages(s"$key$furtherReturnKey$userKey.link", appConfig.furtherReturnWorkOutCGT))</p>
```

Comment:
this shouldn't be read from config, it will eventually be a url you can get from reverse routing. For now, I've put "#" as the href until that page exists